src/hotspot/share/runtime/stubDeclarations.hpp

```diff
@@ -552,6 +552,8 @@
            catch_exception_entry)                                       \
   do_stub(initial, fence)                                               \
   do_entry(initial, fence, fence_entry, fence_entry)                    \
+  do_stub(initial, atomic_add)                                         \
+  do_entry(initial, atomic_add, atomic_add_entry, atomic_add_entry)  \
```

Comment:
Indenting for trailing `\` is not tidy.

Reply:
yes. will update soon. Thanks for pointing this out.

Reply:
updated in the latest commit. thanks.